allennlp/service/servable/models/simple_tagger.py

```diff
@@ -0,0 +1,61 @@
+import codecs
+import os
+from tempfile import gettempdir
+
+from allennlp.common import Params
+from allennlp.data.dataset_readers.sequence_tagging import SequenceTaggingDatasetReader
+from allennlp.data.fields import TextField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.models.simple_tagger import SimpleTagger
+from allennlp.service.servable import Servable, JSONDict
+
+# Simple Tagger Model
+class SimpleTaggerServable(Servable):
```

Comment:
I think eventually we'll just want a single one of these, that loads a `Model` `from_params()`.